__tests__/components/editor/ResourceTemplate.test.js

```diff
@@ -13,14 +13,14 @@ describe('<ResourceTemplate />', () => {
   const wrapper = shallow(<ResourceTemplate.WrappedComponent resourceTemplateId="resourceTemplate:bf2:Note"
                                                              resourceTemplate={reduxState}
                                                              retrieveResourceTemplate={jest.fn()}
-                                                             title="BF2 Work"/>)
+                                                             resourceTemplateLabel="BF2 Work"/>)
 
   it('has div with class "ResourceTemplate"', () => {
     expect(wrapper.find('div.ResourceTemplate').length).toEqual(1)
   })
 
   it('displays the resource label of the resource template', () => {
-    expect(wrapper.find('h1').text()).toEqual('BF2 Work')
+    expect(wrapper.find('#resourceTemplate h1').text()).toEqual('BF2 Work')
```

Comment:
this selector was made more specific and improves comprehension when reading the code.